crates/uv/tests/tool_install.rs

```diff
@@ -321,7 +321,7 @@ fn tool_install_version() {
 
 /// Test an editable installation of a tool.
 #[test]
-fn tool_install_editable() {
+fn tool_install_editable_x() {
```

Comment:
?

Reply:
Needed to disambiguate the tests when running locally (otherwise this test is a prefix of another), fixed.